WordPress/Classes/ViewRelated/Reader/ReaderShowMenuAction.swift

```diff
@@ -63,10 +63,13 @@ final class ReaderShowMenuAction {
                                                     ReaderFollowAction().execute(with: post,
                                                                                  context: context,
                                                                                  completion: {
-                                                                                    guard let vc = vc as? ReaderStreamViewController else {
-                                                                                        return
+                                                                                    if post.isFollowing {
+                                                                                        vc.dispatchSubscribingNotificationNotice(with: post.blogNameForDisplay(), siteID: post.siteID)
+                                                                                    }
+
+                                                                                    if let vc = vc as? ReaderStreamViewController {
+                                                                                        vc.updateStreamHeaderIfNeeded()
                                                                                     }
```

Comment:
If you like, here's  a more compact version: 
`(vc as? ReaderStreamViewController)?.updateStreamHeaderIfNeeded()`

Reply:
Ha. I considered that but thought it might be a bit too much going on. I can certainly do it though!